packages/_infra/src/serve/lambda.tiler.ts

```diff
@@ -36,23 +39,39 @@ export class LambdaTiler extends Construct {
 
     if (props.staticBucketName) environment[Env.AssetLocation] = `s3://${props.staticBucketName}`;
 
+    const code = lambda.Code.fromAsset(CODE_PATH);
     /**
      * WARNING: changing this lambda name while attached to a alb will cause cloudformation to die
      * see: https://github.com/aws/aws-cdk/issues/8253
      */
-    this.lambda = new lambda.Function(this, 'Tiler', {
+    this.lambdaVpc = new lambda.Function(this, 'Tiler', {
       vpc: props.vpc,
       runtime: lambda.Runtime.NODEJS_16_X,
       memorySize: 2048,
       timeout: Duration.seconds(60),
       handler: 'index.handler',
-      code: lambda.Code.fromAsset(CODE_PATH),
+      code,
+      environment,
+      logRetention: RetentionDays.ONE_MONTH,
+    });
+
+    /**
+     * While moving to function URLS create two separate lambda functions
```

Comment:
I'm not sure this is safe to expose lambda directly to the internet?

Reply:
The lambda's are already directly exposed to the internet via a ALB?

Reply:
But I think VPC is the place for us to limit the access.